src/ui/public/vis/editors/config/editor_config_providers.ts

```diff
@@ -54,30 +54,73 @@ class EditorConfigProviderRegistry {
     return this.mergeConfigs(configs);
   }
 
+  private isBaseParam(config: EditorParamConfig): config is NumericIntervalParam {
+    return config.hasOwnProperty('base');
+  }
+
+  private isFixedParam(config: EditorParamConfig): config is FixedParam {
+    return config.hasOwnProperty('fixedValue');
+  }
+
+  private mergeHidden(current: EditorParamConfig, merged: EditorParamConfig): boolean {
+    return Boolean(current.hidden || merged.hidden);
+  }
+
+  private mergeFixedAndBase(
+    current: EditorParamConfig,
+    merged: EditorParamConfig,
+    paramName: string
+  ): { fixedValue?: any; base?: number } {
+    if (
+      this.isFixedParam(current) &&
+      this.isFixedParam(merged) &&
+      current.fixedValue !== merged.fixedValue
+    ) {
+      // In case multiple configurations provided a fixedValue, these must all be the same.
+      // If not we'll throw an error.
+      throw new Error(`Two EditorConfigProviders provided different fixed values for field ${paramName}:
+          ${merged.fixedValue} !== ${current.fixedValue}`);
+    } else if (
+      (this.isFixedParam(current) && this.isBaseParam(merged)) ||
+      (this.isBaseParam(current) && this.isFixedParam(merged))
+    ) {
+      // In case one config tries to set a fixed value and another setting a base value,
+      // we'll throw an error. This could be solved more elegantly, by allowing fixedValues
+      // that are the multiple of the specific base value, but since there is no use-case for that
+      // right now, this isn't implemented.
+      throw new Error(`Tried to provide a fixedValue and a base for param ${paramName}.`);
+    } else if (this.isBaseParam(current) && this.isBaseParam(merged)) {
+      // In case both had where interval values, just use the least common multiple between both interval
+      return {
+        base: leastCommonMultiple(current.base, merged.base),
+      };
+    } else {
```

Comment:
This doesn't need to be inside a condition at all, since the previous conditions all throw or return.